testsuite/features/support/commonlib.rb

```diff
@@ -328,3 +328,100 @@ def add_context(key, value)
   $context[$feature_scope] = {} unless $context.key?($feature_scope)
   $context[$feature_scope].merge!({ key => value })
 end
+
+# This function gets the system name, as displayed in systems list
+# * for the usual clients, it is the full hostname, e.g. suma-41-min-sle15.tf.local
+# * for the PXE booted clients, it is derived from the branch name, the hardware type,
+#   and a fingerprint, e.g. example.Intel-Genuine-None-d6df84cca6f478cdafe824e35bbb6e3b
+def get_system_name(host)
+  case host
+    # The PXE boot minion and the terminals are not directly accessible on the network,
+    # therefore they are not represented by a twopence node
+  when 'pxeboot_minion'
+    output, _code = get_target('server').run('salt-key')
+    system_name =
+      output.split.find do |word|
+        word =~ /example.Intel-Genuine-None-/ || word =~ /example.pxeboot-/ || word =~ /example.Intel/ || word =~ /pxeboot-/
```

Comment:
I want to alert that we don't always have pxeboot written, nor Intel. I'd simplify this to just start with `example.` or `pxeboot` - unless I misunderstood something
![image](https://github.com/uyuni-project/uyuni/assets/1148556/c24c6528-5b30-48c4-91c1-ae917f68cc63)

Reply:
> I want to alert that we don't always have pxeboot written, nor Intel. I'd simplify this to just start with `example.` or `pxeboot` - unless I misunderstood something
> ![image](https://github.com/uyuni-project/uyuni/assets/1148556/c24c6528-5b30-48c4-91c1-ae917f68cc63)
> 

I might just rely too much in my IDE, fixing rubocop style warnings, I will review that, thanks.

Reply:
It seems we handle those in next cases, in any case, I will not refactor this in this PoC.